docs/building-spark.md

```diff
@@ -138,6 +138,13 @@ To avoid the overhead of launching sbt each time you need to re-compile, you can
 in interactive mode by running `build/sbt`, and then run all build commands at the command
 prompt.
 
+### Setting up SBT's Memory Usage
+Configure the JVM options for SBT in `.jvmopts`, for example:
+
+    $ cat .jvmopts                                                                                                                   [21:18:11]
+    -Xms512M
```

Comment:
I doubt we need to suggest `-Xms`; for simplicity, remove it? Also let's make it consistent with the Maven opts above: `-Xmx2g`?

I'm not sure what [21:18:11] is; we don't need to show `cat`. But this does need to say that the file should live in the project root and show its one line of content.

Reply:
Sorry, I committed my console time.